### PR TITLE
DOC-1559: Updated migration guide

### PR DESCRIPTION
**Pull request by TheSpyder**
Related Ticket: DOC-1559

Description of Changes:
* Just comments from the previous PR at this stage

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] ~`_data/nav.yml` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: modules/ROOT/pages/migration-from-5x.adoc [hunk 1]
```diff
@@ -202,14 +203,14 @@ NOTE: Using `text_patterns: []` to turn `text_patterns` off is not supported. It
 
 After upgrading:
 
-* rename any options in your {productname} init configuration to match the new name.
-* remove `textpattern` from your plugins list.
+* Rename any options in your {productname} init configuration to match the new name.
+* Remove `textpattern` from your plugins list.
 
 === `tinymce.activeEditor.options.get`
```
Comment (lnewson) [inline modules/ROOT/pages/migration-from-5x.adoc:209]:
```suggestion
=== `plugins` option
```
Maybe to match the changes @metricjs made in the other PR?

File: modules/ROOT/pages/migration-from-5x.adoc [hunk 2]
```diff
@@ -37,17 +37,7 @@ For additional details on {productname} 6.0 changes, see xref:6.0-release-notes-
...
-Previously, using `closeButton` to disable the close button presented as part of a _Notification_ required a `timeout` value to also be set.
-
-In {productname} 6, `closeButton` shows or hides the close button regardless of a `timeout` value being set or not set.
-
-Equivalently, setting a `timeout` value sets how long the notification presents before automatically closing, regardless of whether a close button is displayed or not.
-
-NOTE: This does not change the behaviour of existing _Notifications_. An `editor.notificationManager` with both `timeout: 5000` and `closeButton: false` set will still present a notification without a close button that automatically closes after displaying for 5 seconds.
-
-=== `disabled` functions renamed in all {productname} APIs
+=== `disabled` wording renamed in all {productname} APIs
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/migration-from-5x.adoc:40]:
```suggestion
=== `disabled` property renamed in all {productname} APIs
```

File: modules/ROOT/pages/migration-from-5x.adoc (hunk 1 above)
Comment (TheSpyder) [reply modules/ROOT/pages/migration-from-5x.adoc:209]: It needs to be a sub-heading of plugins, too.

File: modules/ROOT/pages/migration-from-5x.adoc (hunk 2 above)
Comment (metricjs) [reply modules/ROOT/pages/migration-from-5x.adoc:46]:
```suggestion
=== `disabled` renamed in all {productname} APIs
```
not quite sure on the context, but this honestly might be easier?

**Review by tiny-ben-tran — APPROVED** [review]

File: modules/ROOT/pages/migration-from-5x.adoc [hunk 3]
```diff
@@ -243,23 +243,27 @@ NOTE: Leaving references to removed plugins in a plugin configuration will cause
 
 See below for specific documentation of potential migration issues, if any, regarding these removed plugins.
 
-Seven `plugins` were removed because their functionality is now a Core part of {productname} 6.0:
+Four plugins were removed as per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[{productname} 5 announcement, window="_blank"]:
+
+. `fullpage`
+. `bbcode`
+. `legacyoutput`
+. `tabfocus`
```
Comment (metricjs) [inline modules/ROOT/pages/migration-from-5x.adoc:251]: Maybe copy the note from elsewhere about using tabindex instead?

File: modules/ROOT/pages/migration-from-5x.adoc (hunk 2 above)
Comment (TheSpyder) [reply modules/ROOT/pages/migration-from-5x.adoc:46]: I used wording both here and below, I think the headings should refer to the wording

File: modules/ROOT/pages/migration-from-5x.adoc [hunk 4]
```diff
@@ -435,7 +435,7 @@ As of {productname} 6, the `change` event is not fired unless and until
 
 NOTE: It is still possible to listen for modification to a {productname} editor instance before focus is changed. To do this, listen for the `dirty` event, rather than the `change` event.
 
-=== `disabled` properties renamed to `.enabled` in {productname} UI classes
+=== `.disabled` wording renamed to `.enabled` in {productname} UI classes
```
Comment (metricjs) [inline modules/ROOT/pages/migration-from-5x.adoc:435]:
```suggestion
=== `.disabled` renamed to `.enabled` in {productname} UI classes
```
maybe?

File: modules/ROOT/pages/migration-from-5x.adoc [hunk 5]
```diff
@@ -515,9 +515,9 @@ First, {productname} 6.0 does not alter existing documents. Extant {productname}
 
 If an extant {productname} document with an aligned table is opened using {productname} 6.0 and the *alignment* of the table is re-set, {productname} 6.0 will remove the `float` property and use the `margin` property to set the *alignment*.
 
-Second, if the previous behaviour is required, a custom `format` will need to be created. Specifically, the following CSS (the https://github.com/tinymce/tinymce/blob/release/5.10/modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts#L27-L117[table-specific CSS] from {productname} 5.x’s `DefaultFormats.ts`) must be added to the `format` configuration:
+Second, if the previous behaviour is required, a custom `format` will need to be created. Specifically, the following format (the https://github.com/tinymce/tinymce/blob/release/5.10/modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts#L27-L117[table-specific code] from {productname} 5.x's `DefaultFormats.ts`) must be added to the `format` configuration:
```
Comment (metricjs) [inline modules/ROOT/pages/migration-from-5x.adoc:515]:
```suggestion
Second, if the previous behavior is required, a custom `format` will need to be created. Specifically, the following format (the https://github.com/tinymce/tinymce/blob/release/5.10/modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts#L27-L117[table-specific code] from {productname} 5.x's `DefaultFormats.ts`) must be added to the `format` configuration:
```

**Review by metricjs — APPROVED** [review]

**Review by lnewson — APPROVED** [review]

File: modules/ROOT/pages/migration-from-5x.adoc (hunk 3 above)
Comment (TheSpyder) [reply modules/ROOT/pages/migration-from-5x.adoc:251]: Make a note to do later, I think

**Pull request merged**